nva-datamodel-java/src/main/java/no/unit/nva/model/Funding.java

```diff
@@ -10,10 +11,13 @@
 public class Funding {
 
     private URI source;
+    private URI id;
     private String identifier;
-    private String mainTitle;
-    private Map<String, String> alternativeTitles;
-    private MonetaryAmount amount;
+    private String name;
+    private Map<String, String> alternativeName;
+    private MonetaryAmount fundingAmount;
+    private Instant activeFrom;
+    private Instant activeTo;
```

Comment:
Quick question: The funding itself has an @id, which entails that every property relates to the object dereferenced by the @id. Is this the case, such that the @id always and only has the same values for e.g. funding amount, active*?

Reply:
Not really. If source is NFR, activeFrom and activeTo can be resolved by the `id`. fundingAmount is something that can be set per use of the funding, e.g. to indicate that a funding is split on different projects. Other verified funding sources we integrate to in the future might not even have `activeFrom` and `activeTo`.

Reply:
I think that from publication api perspective, a funding should have either an `id` or a `name`. The rest of the fields should be optional. `name` is used when the data is entered manually in the form.

Reply:
Another aspect worth mentioning is that I have added source URI to verified fundings, so the `source` is also given if an `id` is provided.

Reply:
@brinxmat Do we want to change something based on this insight?